tests/e2e/features/dicomViewer.feature

```diff
@@ -1,12 +1,12 @@
 Feature: Preview dicom image
-  As a user
-  I want to preview a DICOM file
-  so that I can preview DICOM image and its metadata
+  As a user who has a DICOM file
+  I want to preview the file
+  so that I can reassure that it contains the right image and check its metadata
```

Comment:
```suggestion
  so that I can be reassured that it contains the right image and check its metadata
```

```suggestion
  so that I can assure myself that it contains the right image and check its metadata
```

```suggestion
  so that I can see that it contains the right image and check its metadata
```

A few alternatives.

Reply:
thanks for the suggestions